applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.h

```diff
@@ -55,50 +53,44 @@ namespace Kratos
 /// Utility to modify497 the distances of an embedded object in order to avoid bad intersections
 /// Besides, it also deactivate the full negative distance elements
 
-class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MassConservationCheckProcess : public Process
+class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MassConservationUtility
 {
 public:
     ///@name Type Definitions
     ///@{
 
     /// Pointer definition of DistanceModificationProcess
-    KRATOS_CLASS_POINTER_DEFINITION(MassConservationCheckProcess);
+    KRATOS_CLASS_POINTER_DEFINITION(MassConservationUtility);
 
     typedef Node<3> NodeType;
     typedef Geometry<NodeType> GeometryType;
 
+
     ///@}
     ///@name Life Cycle
     ///@{
 
     /**
      * @brief Constructor with separate paramters
      *
-     * @param rModelPart Complete model part (including boundaries) for the process to operate on
-     * @param PerformCorrections Choice if only logging is required (false) or if corrections by shifting the distance field shall be performed (true)
+     * @param rModelPart Complete model part (including boundaries) for the utility to operate on
      * @param CorrectionFreq Frequency of the correction (if wished) in time steps
-     * @param WriteToLogFile Choice if results shall be written to a log file in every time step
-     * @param LogFileName Name of the log file (if wished)
      */
-    MassConservationCheckProcess(
-        ModelPart& rModelPart,
-        const bool PerformCorrections,
-        const int CorrectionFreq,
-        const bool WriteToLogFile,
-        const std::string LogFileName);
+    MassConservationUtility(
+        ModelPart& rModelPart);
```

Comment:
Why do we need a constructor without settings?

Reply:
removed and added  constructor with model